hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ClusterWideConfigurationService.java

```diff
@@ -271,6 +277,15 @@ public void registerConfigLocally(IdentifiedDataSerializable newConfig,
         } else if (newConfig instanceof LockConfig) {
             LockConfig lockConfig = (LockConfig) newConfig;
             currentConfig = lockConfigs.putIfAbsent(lockConfig.getName(), lockConfig);
+        } else if (newConfig instanceof AtomicLongConfig) {
```

Comment:
Why didn't we see any failures for the missing `AtomicLongConfig` and `AtomicReferenceConfig` before?  Are we missing a test?

Reply:
That's a good question to @vbekiaris but it's out of scope of this PR, so plz address any changes related to it in another PR.